benefits/core/migrations/0002_data.py

```diff
@@ -156,6 +161,26 @@ def load_data(app, *args, **kwargs):
         enrollment_success_expiry_item_details=None,
     )
 
+    mst_veteran_verifier = EligibilityVerifier.objects.create(
+        name=os.environ.get("MST_VETERAN_VERIFIER_NAME", "VA.gov - Veteran (MST)"),
+        eligibility_type=mst_veteran_type,
+        auth_provider=auth_provider,
+        selection_label=_("eligibility.pages.index.veteran.label"),
+        selection_label_description=_("eligibility.pages.index.veteran.description"),
+        start_title=_("eligibility.pages.start.veteran.title"),
+        start_headline=_("eligibility.pages.start.veteran.headline"),
+        start_item_heading=_("eligibility.pages.start.veteran.start_item.heading"),
+        start_item_details=_("eligibility.pages.start.veteran.start_item.details"),
+        start_item_secondary_details=_("eligibility.pages.start.veteran.start_item.secondary_details"),
+        unverified_title=_("eligibility.pages.unverified.login_gov.title"),
+        unverified_blurb=_("eligibility.pages.unverified.login_gov.p[0]"),
```

Comment:
I wonder if we should rename this PO entry since it is now being used for the Login.gov `EligibilityVerifier` and the veteran `EligibilityVerifier`? 

Also, I'm looking at how we have a [different PO entry being used for Courtesy Cards](https://github.com/cal-itp/benefits/pull/1436/files#diff-f4aa1ce24b8c65ccf8b80dacab3cb3aff3e95d9bf430e8b22bc66eb233f40b23R215) ([`eligibility.pages.unverified.mst_cc.title`](https://github.com/cal-itp/benefits/blob/dev/benefits/locale/en/LC_MESSAGES/django.po#L121)), but... does the "Developer Benefits Full App" page in Figma imply now that all `unverified_title` and `unverified_blurb`s should be the same?

![image](https://github.com/cal-itp/benefits/assets/25497886/46fbe358-1690-40ea-8d9b-0291dd44aca3)

![image](https://github.com/cal-itp/benefits/assets/25497886/49f191d5-9aa7-497d-a468-78eaaad8d1a4)


cc @srhhnry @thekaveman

Reply:
Side thought -- this just gave me an idea for something to try during dev-design handoff (I worry I'll forget it if I wait until retro): 

As a part of design-dev handoff, could we go ahead and make GitHub tickets at that point to capture the changes that need to be done?

Reply:
Yeah and now the enrollment page copy is all the same - so there's some more copy string consolidation that can be done. But not sure what's in-scope vs. out-of-scope in this ticket and in this Sprint overall.